code/modules/antagonists/revolution/revolution.dm

```diff
@@ -163,6 +166,16 @@
 	var/give_flash = FALSE
 	var/give_hud = TRUE
 
+/datum/antagonist/rev/head/on_mindshield(mob/implanter)
+	if(!silent)
+		owner.current.visible_message(
+			span_warning("[owner.current] seems to resist the implant!"),
+			span_warning("You feel something interfering with your mental conditioning, but you resist it!")
+		)
+	for(var/obj/item/implant/mindshield/mindshield_implant in owner.current.implants)
+		mindshield_implant.removed(owner.current, TRUE)
+		qdel(mindshield_implant)
```

Comment:
It'd be nicer for this bit to be separate from the other reactions to mindshields. So you could have something like COMSIG_PRE_IMPLANT and then if the right bitflag is returned, print the message about failing to convert and then return in the main minefield proc. Seems cleaner and less prone to pain to me

Reply:
I hope I did this properly